api/valor_api/backend/core/llm_clients.py

```diff
@@ -163,15 +166,12 @@ def __call__(
                 messages=processed_messages,
                 seed=self.seed,
             )
-        # TODO Should we catch this if we aren't going to do anything?
-        except openai.BadRequestError as e:
+        except Exception as e:
```

Comment:
I made this more generic since BadRequestError is hard to test (requires extra inputs like `response` and `body`)